decidim-proposals/lib/tasks/decidim_proposals_tasks.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+namespace :decidim_proposals do
+  desc "Set a published_at value to each proposal from the updated_at value"
+  task copy_updated_at_to_published_at: :environment do
+    Decidim::Proposals::Proposal.all.each do |proposal|
```

Comment:
Use `find_each` instead of `each`

Reply:
Although I think this would just work:

`Decidim::Proposals::Proposal.update_all("published_at = updated_at")`